cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -236,7 +234,9 @@ public void drop(String name) throws Exception {
 
   @Override
   public void upgrade(String tableName, Properties properties) throws Exception {
```

Comment:
Otherwise the one who calls upgrade needs to know what table is a queue table, which is wrong. The queue table(s) should be encapsulated in this class, not exposed.